style.css

```diff
@@ -0,0 +1,1404 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+}
+
+@media only screen and (min-width: 768px) {
+  body {
+    background-color: white;
+  }
+
+  .texts2 {
+    display: none;
+  }
+
+  .footer-desk-mob {
+    display: none;
+  }
+
+  .welcome-about2 {
+    display: none;
+  }
+
+  .linejs {
+    border: 1px solid gray;
+    width: 15%;
+    margin-top: 15px;
+  }
+
+  .header-nav {
+    background-color: dimgrey;
+    display: flex;
+    height: 50px;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  .program a {
+    text-decoration: underline;
+    font-size: 20px;
+    font-weight: 400;
+    margin-top: 25px;
+    color: #fff;
+  }
+
+  .nav-bar a {
+    text-decoration: none;
+    color: black;
+    font-family: "Lato", sans-serif;
+    font-weight: 400;
+    font-size: 30px;
+  }
+
+  .brands a {
+    color: rgba(192, 192, 192, 0.3);
+    font-size: 30px;
+    font-weight: 700;
+    text-decoration: none;
+  }
+
+  .header-nav ul a {
+    padding-right: 20px;
+  }
+
+  .fab {
+    color: white;
+  }
+
+  .icons {
+    display: flex;
+
+    /* gap: 20px; */
```

Comment:
- 🙏Please remove commented code from your project as it contributes to keeping your codebase clean🗑 and organized. Commented code serves no purpose in the final product and can make it difficult😣 for other developers to understand and maintain the codebase effectively.